docs/workflows/input_output.md

```diff
@@ -543,8 +540,8 @@ the resultant output would look like:
 }
 ```
 
-## Additional Features
+## Additional Feature
```

Comment:
One might argue this header is unneeded, open to suggestions to an alternate location for this info.

Reply:
I think it's fine to leave it as `Additional features`, even if there is only one right now. I'm not sure where a better location would be

Reply:
```suggestion
## Additional features
```